modules/apps/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/edit_form/start.jsp

```diff
@@ -18,7 +18,7 @@ String fullName = namespace.concat(HtmlUtil.escapeAttribute(name));
 
 		<div aria-orientation="vertical" class="panel-group panel-group-flush" role="tablist">
 			<c:if test="<%= Validator.isNotNull(onSubmit) %>">
-				<div aria-label="<%= HtmlUtil.escape(portletDisplay.getTitle()) %>" class="input-container" role="group">
+				<div aria-label="<%= HtmlUtil.escape(portletDisplay.getTitle()) %>" class="input-container" id="<%= namespace %>groupLabel" role="group">
```

Comment:
`groupLabel` is a pretty generic ID, for an element that's visible in DOM and accessed globally. Can we call this something more specific, like `id="<%= namespace %>editFormInputContainer"`.